modules/pam_limits/pam_limits.c

```diff
@@ -816,9 +816,22 @@ parse_config_file(pam_handle_t *pamh, const char *uname, uid_t uid, gid_t gid,
         pam_syslog(pamh, LOG_DEBUG, "reading settings from '%s'", CONF_FILE);
     fil = fopen(CONF_FILE, "r");
     if (fil == NULL) {
-        pam_syslog (pamh, LOG_WARNING,
-		    "cannot read settings from %s: %m", CONF_FILE);
-        return PAM_SERVICE_ERR;
+      int err = errno;
+#ifdef VENDORDIR
+
+      /* if the specified file does not exist, and it is not provided by
+	 the user, try the vendor file as fallback. */
+      if (pl->conf_file == NULL)
+	fil = fopen(VENDORDIR"/security/limits.conf", "r");
+
+      if (fil == NULL)
+#endif
+	{
+	  pam_syslog (pamh, LOG_WARNING,
+		      "cannot read settings from %s: %s", CONF_FILE,
+		      strerror(err));
+	  return PAM_SERVICE_ERR;
+	}
```

Comment:
The indentation seems to be somehow weird here.

Reply:
If you remove the #if/#endif block, the indentation looks correct.
Since this part is outside of the if/endif block: I can let it as it is or adjust the indentation to the if/endif block. I'm fine with both.

Reply:
IMO the indentation should be correct with the conditionally compiled block present.